### PR TITLE
Added Swedish Language

### PR DESCRIPTION
**Pull request by edlaika**
Added support for the Swedish Language. The purpose is to expand on the application usability with a help of more languages. I have read all the rules and guidelines and followed the standards based on the description.

### CODE REVIEW

File: app/src/main/res/values-sv/strings.xml [hunk 1]
```diff
@@ -0,0 +1,952 @@
...
+    <string name="tip_more_actions_text">Tryck här för fler alternativ, t.ex. visning av liknande. \ n \ nTryck här för att avvisa</string>
+    <string name="tip_compose_message_title">Skriv meddelandet</string>
+    <string name="tip_compose_message_text">Skicka ett meddelande till den nuvarande användaren. \n\ nTryck här för att avvisa</string>
+
+    <!--Media attachments on the reply activity-->
+    <string name="attach_media">Infoga media</string>
+    <string name="attach_link_title">Infoga webblänk</string>
+    <string name="attach_link_text">Ange webbadress t.ex. https://anilist.co/forum/thread/2459/1</string>
+    <string name="attach_image_title">Infoga bildlänk</string>
+    <string name="attach_image_text"Ange bildadress, t ex: https:/media.giphy.com/media/l41YxdIcDU58r5CDK/giphy.gif</string>
```
Comment (youssefhabri) [inline app/src/main/res/values-sv/strings.xml]: There's a missing `>` in this line
```suggestion
    <string name="attach_image_text">Ange bildadress, t ex: https:/media.giphy.com/media/l41YxdIcDU58r5CDK/giphy.gif</string>
```

Comment (edlaika) [reply app/src/main/res/values-sv/strings.xml]: Sorry did not see that there thanks for having sharp eyes!😊

File: app/src/main/res/values-sv/strings.xml [hunk 2]
```diff
@@ -0,0 +1,952 @@
...
+    <string name="text_multiple_videos">Flera videor</string>
+
+    <string name="text_insufficient_genres">För lite genrer just nu!</string>
+
+    <string name="text_checking_collection">Söker i ditt bibliotek ..</string>
+
+    <string name="text_processing_request">Behandling av förfrågan ..</string>
+    <string name="text_permission_required">AniTrend beviljades inte tillstånd!</string>
+
+    <string name="text_read_more"Läs mer</string>
```
Comment (youssefhabri) [inline app/src/main/res/values-sv/strings.xml]: Missing `>`
```suggestion
    <string name="text_read_more">Läs mer</string>
```

Comment (youssefhabri) [reply app/src/main/res/values-sv/strings.xml]: I've also checked the rest of the file, I haven't spotted anymore issues. Hopefully everything is ok now 😃

Comment (edlaika) [reply app/src/main/res/values-sv/strings.xml]: Thanks for having sharp eyes I think I might need glasses soon.

**Comment by wax911** [comment]
Thanks for the PR @edlaika we always appreciate the extra help!! :heart: and also thanks @youssefhabri :1st_place_medal: one hell of a sharp eye there :smirk:

**Pull request merged**